NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -586,7 +595,8 @@ class NeuroID private constructor(
         val finalSessionID = if (sessionID != null && sessionID != "") {
             sessionID
         } else {
```

Comment:
isn't this just an empty else statement now?

Reply:
I don't think it is an empty if. We need to assign something to it otherwise the setUser() will fail (it is expecting a non-null String). Let me check that.